src/libraries/Directory.Build.targets

```diff
@@ -272,4 +273,20 @@
       </ItemGroup>
     </When>
   </Choose>
+
+  <Target Name="AddLibIntelDocs">
+   <ItemGroup>
+      <TfmSpecificPackageFile Include="$(XmlDocFileRoot)/1033/$(TargetName).xml" Condition="Exists('$(XmlDocFileRoot)/1033/$(TargetName).xml')">
+        <PackagePath>lib/$(TargetFramework)</PackagePath>
+      </TfmSpecificPackageFile>
+    </ItemGroup>
+  </Target>
+
+  <Target Name="RemoveXmlFilesFromBuildOutput" BeforeTargets="GenerateNuspec">
```

Comment:
Is this because we want to use the ones from the intellisense package?

Reply:
yes thats correct.